client/gulpFile.js

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable no-console, no-use-before-define */
+const browserify       = require("browserify")
+const watchify         = require("watchify")
+const gulp             = require("gulp")
+const uglify           = require("gulp-uglify-es").default
+const source           = require("vinyl-source-stream")
+const buffer           = require("vinyl-buffer")
+const htmlReplace      = require("gulp-html-replace")
+const envify           = require('envify/custom')
+const notifier         = require('node-notifier')
+const sass             = require('gulp-sass')
+const clean            = require('gulp-clean')
+const uuidv4           = require('uuid/v4')
+const strip            = require('gulp-strip-comments')
+const concat           = require('gulp-concat')
+
+require("dotenv").config()
+
+/**
+ *    Development tasks
+ */
+
+gulp.task("clean", _cb => {
+    return gulp.src('tmp/', {read: false})
+        .pipe(clean())
+})
+
+gulp.task("sass", cb => {
+    gulp.src(['sass/index.scss', 'node_modules/react-toastify/dist/ReactToastify.css'])
+    .pipe(sass({outputStyle: 'compressed'}).on('error', err => {
+        notifier.notify({
+            title: "Sass Error",
+            message: err.message
+        })
+        cb(err)
+    }))
+    .pipe(concat('index.css'))
+    .pipe(gulp.dest('tmp/static/css'))
+    .on("finish", () => {
+        cb()
+    })
+})
+
+gulp.task("fonts", cb => {
+    gulp.src("static/fonts/**")
+    .pipe(gulp.dest("tmp/static/fonts"))
+    .on("finish", () => {
+        cb()
+    })
+})
+
+gulp.task("js", cb => {
+    let bundler = browserify({
```

Comment:
Why not make straight use of `babel` instead?

Reply:
To be honest I am not sure why I used browserify initially. I think it is because my first project I used JQuery.